unit-tests/func/mur/l500/test-change-sensor-mode.cpp

```diff
@@ -0,0 +1,32 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+//#test:tag max-usable-range
+//#test:device L500*
+
+#include "../../func-common.h"
+
+
+// Test group description:
+//       * This tests group verifies depth sensor max usable range option restrictions,
+//         according to [RS-8358]
+//
+// Current test description:
+//       * While not streaming, activating Max Usable Range option should make sure the sensor mode is 'VGA'
+//         if it is not VGA it will set the sensor mode to VGA
+//       
+// Note: * L515 specific test
+//       * Test will pass if no L515 device is connected
+//       * Test will be performed on the first L515 device detected.
+
+TEST_CASE( "Max Usable Range option set sensor mode if not streaming", "[max-usable-range]" )
```

Comment:
Can't we just combine this test with the other?

Reply:
Combined